app/assets/javascripts/modules/DataRequestActions.js

```diff
@@ -33,11 +33,17 @@ moj.Modules.DataRequestActions = {
   },
 
   clearNoteField: function() {
+    if (!$('#data_request_request_type_bwcf').is(':checked')) {
```

Comment:
`#request_type_note_for_bwcf_panel` and `#request_type_note_for_cctv_panel` are not the correct IDs.

The correct IDs are:
`request_type_bwcf_panel`
`request_type_cctv_panel`